app/views/test_executions/_execution_results.html.erb

```diff
@@ -8,20 +8,17 @@
 
 
 <% if execution.incomplete? %>
-  <h4>
-    <%= "PENDING: #{execution.created_at.to_time.strftime('%b %d, %Y at %I:%M:%S %p (%A)')}" %>
-  </h4>
+  <h4><i class="fa fa fa-circle-o-notch fa-spin inline-icon info-icon"></i>In Progress</h4>
```

Comment:
`fa` is used twice. Also wondering whether the poorly-named `inline-icon` class could be replaced with `fa-fw` and you can insert a space between the `</i>` tag and the text..